run-worker.py

```diff
@@ -0,0 +1,22 @@
+"""Initializes a slim rq task queue. File taken from
+http://python-rq.org/patterns/"""
+
+import os
+import urlparse
+from redis import Redis
+from rq import Worker, Queue, Connection
+
+listen = ['high', 'default', 'low']
```

Comment:
I think it's standard to use these three queues but flask-base might only need the default queue.